source/examples/verovio/LisztMazeppa.mei

```diff
@@ -0,0 +1,388 @@
+                                </layer>
+                            </staff>
+                            <staff xml:id="ex-2028510780" n="2">
+                                <layer xml:id="ex-1815230667" n="5">
+                                    <rest xml:id="ex-1429130987" dur="4" />
+                                    <clef xml:id="ex-1947390461" shape="G" line="2" />
+                                    <chord xml:id="ex-1553574041" dur="8" stem.dir="up">
+                                        <artic xml:id="ex-1812726861" artic="stacciss" />
+                                        <note xml:id="ex-2140697747" oct="4" pname="c">
+                                            <accid xml:id="ex-1913495638" accid="s" />
+                                        </note>
+                                        <note xml:id="ex-1725813982" oct="4" pname="g" />
+                                        <note xml:id="ex-1841459092" oct="4" pname="b" accid.ges="f" />
+                                        <note xml:id="ex-1970967523" oct="5" pname="e" />
+                                    </chord>
+                                    <rest xml:id="ex-1115904086" dur="8" />
+                                    <rest xml:id="ex-1025284151" dur="4" />
+                                    <chord xml:id="ex-0553655856" dur="8" stem.dir="up">
+                                        <artic xml:id="ex-0247329341" artic="stacciss" />
+                                        <note xml:id="ex-0541942329" oct="3" pname="g">
+                                            <accid xml:id="ex-0946576576" accid="s" />
+                                        </note>
+                                        <note xml:id="ex-0983648271" oct="4" pname="d" />
+                                        <note xml:id="ex-0847376091" oct="4" pname="f" />
+                                        <note xml:id="ex-1885898180" oct="4" pname="b">
+                                            <accid xml:id="ex-1579565187" accid="n" />
+                                        </note>
+                                    </chord>
+                                    <rest xml:id="ex-0559253695" dur="8" />
+                                </layer>
+                            </staff>
+                            <tempo xml:id="ex-0484835033" place="above" staff="1" tstamp="1">
+                                <rend xml:id="ex-1069442913" fontweight="bold">Allegro</rend>
+                            </tempo>
+                            <dynam xml:id="ex-0283570703" place="below" staff="1" startid="#ex-0731379794">ff</dynam>
+                            <octave xml:id="ex-0117730886" staff="1" startid="#ex-0731379794" endid="#ex-0579048232" dis="8" dis.place="above" />
+                            <arpeg xml:id="ex-0149852838" plist="#ex-0731379794 #ex-1553574041" />
+                            <arpeg xml:id="ex-1483377242" plist="#ex-1474174387 #ex-0553655856" />
```

Comment:
No edit-start and edit-end snippets in this file?

Reply:
Also, file is not used?

Reply:
Also, encoding here includes the first 4 measures, image used in the guidelines only shows 2 measures?